MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/location/LocationLayerController.java

```diff
@@ -303,10 +313,9 @@ public void onNewAnimationValue(Float newPulseRadiusValue) {
   Set<AnimatorListenerHolder> getAnimationListeners() {
     Set<AnimatorListenerHolder> holders = new HashSet<>();
     holders.add(new AnimatorListenerHolder(MapboxAnimator.ANIMATOR_LAYER_LATLNG, latLngValueListener));
+    holders.add(new AnimatorListenerHolder(MapboxAnimator.ANIMATOR_LAYER_GPS_BEARING, gpsBearingValueListener));
```

Comment:
This will make animator running and computing values even if we're not consuming them. When switching the mode we're not interested in the animated value but rather the final value, so maybe there's a chance that we can push the bearing when updating the mode instead of caching the animation?